src/videouplinkbandwidthpolicy/NScaleVideoUplinkBandwidthPolicy.ts

```diff
@@ -196,13 +198,20 @@ export default class NScaleVideoUplinkBandwidthPolicy implements VideoUplinkBand
       !this.hasBandwidthPriority &&
       this.numParticipants > 2
     ) {
-      const targetHeight =
+      let targetHeight =
         NScaleVideoUplinkBandwidthPolicy.targetHeightArray[
           Math.min(
             this.numParticipants,
             NScaleVideoUplinkBandwidthPolicy.targetHeightArray.length - 1
           )
         ];
+      if (
+        targetHeight === 480 &&
+        this.browserBehavior &&
+        this.browserBehavior.disable480pResolutionScaleDown()
```

Comment:
can this be simplified as `this.browserBehavior?.disable480pResolutionScaleDown()`?

Reply:
Thought about it but then we say we support Node 12+ and optional chaining is only available in Node 14+. Anyway, check the code and it seems we already use it everywhere else so updated :)